cmd/lora/main.go

```diff
@@ -33,7 +32,10 @@ const (
 	defLogLevel       = "error"
 	defHTTPPort       = "8180"
 	defLoraMsgURL     = "tcp://localhost:1883"
-	defSubTimeout     = "30s" // 30 seconds
+	defLoraMsgTopic   = "application/+/device/+/event/up"
+	defLoraMsgUser    = ""
+	defLoraMsgPass    = ""
+	defLoraMsgTimeout = "30s" // 30 seconds
```

Comment:
Consider removing this comment, it is obvious what `30s` means if the variable is called `...Timeout`